build/server-image/kube-apiserver/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
discussed already offline but writing here:

I found that you can do the first stage with `FROM --platform=linux/${HOST_ARCH}` while doing the second stage with `FROM --platform=linux/${TARGETARCH}` without needing to `RUN` on the target arch at all (setcap sets file metadata that is cross platform) so we can do this without qemu if we plumb through the host arch.

that will let the build be less priv and avoid fighting with qemu / s390x etc.